readme.markdown

```diff
@@ -261,6 +261,17 @@ To run tests a single time, simply run the following command. This is used in CI
 npm test
 ```
 
+# Troubleshooting
+
+### CSS Classes
+
+Dragula uses only four CSS classes. Their purpose is quickly explained below but you can check dist/dragula.css to see the corresponding CSS rules.
+
+* **.gu-unselectable** is added to the DOM's body when dragging. It prevents inline elements to be selected by the dragging.
```

Comment:
"It prevents inline elements to be selected by the dragging." -> "You can use it to style the body while something is being dragged."

"DOM's body" -> "`document.body` element"